apps/etterlatte-utbetaling/src/main/kotlin/no/nav/etterlatte/utbetaling/iverksetting/utbetaling/Utbetaling.kt

```diff
@@ -185,6 +186,7 @@ enum class OppdragKlassifikasjonskode(
                 "TBTREKK" -> TBTREKK
                 "UTBETALING" -> UTBETALING
                 "FRISSKAT" -> FRISSKAT
+                "BPSKSKAT" -> BPSKSKAT
```

Comment:
Skulle dette være med her?

Reply:
nei, aner ikke hvordan det kom inn her. Litt redd for at det blir borte om jeg fjerner det men tester